test/test_juicebox_assembly_converter.py

```diff
@@ -0,0 +1,195 @@
+#!usr/bin/python
+'''
+Shawn Sullivan
+October 31, 2018
+Phase Genomics
+
+juicebox_scripts/test_juicebox_assembly_converter.py
+
+This file contains unit tests for functions of the juicebox_assembly_converter.py script.
+
+Copyright 2018, Phase Genomics Inc. All rights reserved.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see https://www.gnu.org/licenses/agpl-3.0.en.html
+'''
+
+import filecmp
+import os
+import unittest
+#uncomment the below lines if you need to have the ability to run this
+#file directly either from the the root directory, but you can avoid
+#that by running `python -m unittest discover` within the project root
+#directory instead of trying to call this file directly
+#import sys
+# sys.path.append('src/')
+# sys.path.append('../src/')
+from juicebox_assembly_converter import JuiceboxConverter, ProcessedAssembly
+from juicebox_assembly_converter import InvalidFastaError, MissingFragmentError, UnscaffoldedContigError, ZeroLengthContigError 
+
+class JuiceboxConverterTestCase(unittest.TestCase):
+    def setUp(self):
+        self.converter = JuiceboxConverter()
+        self.test_output_fasta = 'test_output.fasta'
+        self.test_output_agp = 'test_output.agp'
+        self.test_output_bed = 'test_output.bed'
+        self.test_output_break_report = 'test_output.break_report.txt'
+        
+        self.collateral_dir = 'test/collateral/'
```

Comment:
Build an absolute path from the __file__ variable like this:
`os.path.dirname(__file__) + '/test/collateral/`